app/views/register/suitability/NonTaxableTrustRegistrationAccessCodeView.scala.html

```diff
@@ -35,7 +35,7 @@
 
         @components.error_summary(form.errors)
 
-        @components.heading("nonTaxableTrustRegistrationAccessCode.heading")
+        <p>@components.heading_with_caption("nonTaxableTrustRegistrationAccessCode")</p>
```

Comment:
Don't believe this needs to be inside a <p> element

Reply:
Updated